react/src/views/Boxes/BoxesView.test.tsx

```diff
@@ -34,7 +34,7 @@ describe("Boxes view", () => {
                         gender: "Women",
                         name: "Long Dress",
                       },
-                      items: 64,
+                      itnumberOfItemsems: 64,
```

Comment:
This doesnt look right :laughing:

Reply:
You passed the test! #weeklyPrReviewQualityAssurance